RequestService/RequestService.Repo/ApplicationDbContext.cs

```diff
@@ -56,7 +49,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            modelBuilder.Query<DailyReport>().ToQuery(() => DailyReport.FromSql("TwoHourlyReport"));
+            //modelBuilder.Query<DailyReport>().ToQuery(() => DailyReport.FromSql("TwoHourlyReport"));
```

Comment:
Commenting intentional?

Reply:
Intentional. I need your help in trying to figure out how to get this to work with NetCoreV3. The same problem exists in the UserService and the RequestService too.